tests/trinity/integration/fixtures/20pow_headers.ldb/CURRENT

```diff
@@ -0,0 +1 @@
+MANIFEST-000027
```

Comment:
What do you think about zipping this directory and to add unzipping it as part of the CI setup.  We should only have to do the unzip once as a `session` scoped fixture, and then we can do the directory copy from there. 

- It cleans up the git history a bit by not having each of the leveldb files laying around
- It prevents us *accidentally* connecting to and modifying the canonical database.